### PR TITLE
add github_dark_dimmed theme

### PR DESCRIPTION
**Pull request by fabianocouto**
## Preview with Github Dark Dimmed
image

## Preview with Github Default
image

### CODE REVIEW

**Comment by Zo-Bro-23** [comment]
I like this - @rickstaa?

File: themes/index.js [hunk 1]
```diff
@@ -416,6 +416,13 @@ export const themes = {
     border_color: "85A4C0",
     bg_color: "030314",
   },
+  github_dark_dimmed: {
```
Comment (rickstaa) [inline themes/index.js:419]: Maybe let's place this under the `github_dark` theme?

https://github.com/anuraghazra/github-readme-stats/blob/542a2326ad2cfe03a7bc36d5cb75dc86c4da3b70/themes/index.js#L318-L323

**Review by rickstaa — CHANGES_REQUESTED** [review]

**Comment by rickstaa** [comment]
> I like this - @rickstaa?

Great theme! I would like to merge this. I, however, think it would be nice to keep it close to the other GitHub themes. After that change, we are good to go.

File: themes/index.js (hunk 1 above)
Comment (fabianocouto) [reply themes/index.js:419]: I put it at the end of the file because of the contribution guidelines. But I agree with the suggestion. ;)

**Review by rickstaa — APPROVED** [review]

**Comment by vishalsingh2972** [comment]
@fabianocouto  I want the same in this theme, also instead of "A" or"A++" I want it to be like this with the github logo
image

Tried a lot but unable to get it could you please help me with this!

**Comment by vishalsingh2972** [comment]
@fabianocouto  I want the same in this theme, also instead of "A" or "A++" I want it to be like this with the github logo



Tried a lot but unable to get it could you please help me with this!

**Comment by vishalsingh2972** [comment]
@fabianocouto  I want the same in this theme, also instead of "A" or "A++" I want it to be like this with the github logo



Tried a lot but unable to get it could you please help me with this!

**Comment by fabianocouto** [comment]
> @fabianocouto I want the same in this theme, also instead of "A" or "A++" I want it to be like this with the github logo
> 
> 
> 
> Tried a lot but unable to get it could you please help me with this! @fabianocouto @Zo-Bro-23 @rickstaa @anuraghazra

Hi, @vishalsingh2972!
I don't know if I understood your idea very well.
But are you suggesting something like that?

I added a new option called `rank_icon` with initially two possible values: `default` or `github`.

```
https://fabianocouto-readme-stats.vercel.app/api?username=fabianocouto&show_icons=true&include_all_commits=true&count_private=true&theme=github_dark_dimmed&hide=stars&line_height=28&rank_icon=github
```

fabianocouto

fabianocouto

fabianocouto

**Comment by vishalsingh2972** [comment]
> > @fabianocouto I want the same in this theme, also instead of "A" or "A++" I want it to be like this with the github logo
> > 
> > Tried a lot but unable to get it could you please help me with this! @fabianocouto @Zo-Bro-23 @rickstaa @anuraghazra
> 
> Hi, @vishalsingh2972! I don't know if I understood your idea very well. But are you suggesting something like that?
> 
> I added a new option called `rank_icon` with initially two possible values: `default` or `github`.
> 
> ```
> https://fabianocouto-readme-stats.vercel.app/api?username=fabianocouto&show_icons=true&include_all_commits=true&count_private=true&theme=github_dark_dimmed&hide=stars&line_height=28&rank_icon=github
> ```
> 
> fabianocouto
> 
> fabianocouto
> 
> fabianocouto

yes @fabianocouto  I wanted this in green colour with the github logo, also the streak card in same green color!

**Comment by vishalsingh2972** [comment]
@fabianocouto  needed this in green with the github logo instead of A++
<div>
  <a title="GitHub Readme Stats">
  </a>
  <a title="Github Readme Streak Stats">
  </a>
</div>

**Comment by vishalsingh2972** [comment]
@fabianocouto  basically I need the below one to be in format of green like in https://github.com/anuraghazra/github-readme-stats/pull/2594#issuecomment-1510435964 with the github logo instead of A+,  also the streak card in same green color!
<div>
  <a title="GitHub Readme Stats">
  </a>
  <a title="Github Readme Streak Stats">
  </a>
</div>

**Comment by fabianocouto** [comment]
@vishalsingh2972 you can use the options to customize the theme colors.
https://github.com/anuraghazra/github-readme-stats#common-options
https://github.com/denvercoder1/github-readme-streak-stats#-options

```html
<div class="d-block">
  <a href="https://github.com/anuraghazra/github-readme-stats" title="GitHub Readme Stats"><img src="https://fabianocouto-readme-stats.vercel.app/api?username=vishalsingh2972&show_icons=true&include_all_commits=true&count_private=true&theme=github_dark_dimmed&hide=stars&line_height=28&v1&rank_icon=github&text_color=7cfe9e&title_color=7cfe9e&icon_color=7cfe9e" alt="vishalsingh2972" width=49.5% /></a>
  <a href="https://github.com/denvercoder1/github-readme-streak-stats" title="Github Readme Streak Stats"><img src="https://github-readme-streak-stats.herokuapp.com/?user=vishalsingh2972&background=24292f&border=373e47&stroke=373e47&currStreakNum=adbac7&sideNums=adbac7&sideLabels=adbac7&dates=adbac7&ring=7cfe9e&currStreakLabel=7cfe9e&fire=7cfe9e" alt="vishalsingh2972" width=49.5% /></a>
</div>
```

<div class="d-block">
  <a href="https://github.com/anuraghazra/github-readme-stats" title="GitHub Readme Stats">vishalsingh2972</a>
  <a href="https://github.com/denvercoder1/github-readme-streak-stats" title="Github Readme Streak Stats">vishalsingh2972</a>
</div>

**Comment by vishalsingh2972** [comment]
> @vishalsingh2972 you can use the options to customize the theme colors. https://github.com/anuraghazra/github-readme-stats#common-options https://github.com/denvercoder1/github-readme-streak-stats#-options
> 
> ```
> <div class="d-block">
>   <a href="https://github.com/anuraghazra/github-readme-stats" title="GitHub Readme Stats">vishalsingh2972</a>
>   <a href="https://github.com/denvercoder1/github-readme-streak-stats" title="Github Readme Streak Stats">vishalsingh2972</a>
> </div>
> ```
> 
> [vishalsingh2972](https://github.com/anuraghazra/github-readme-stats) [vishalsingh2972](https://github.com/denvercoder1/github-readme-streak-stats)

Thank You so much @fabianocouto 💯 :)

**Comment by rickstaa** [comment]
> > @fabianocouto I want the same in this theme, also instead of "A" or "A++" I want it to be like this with the github logo
> > 
> > Tried a lot but unable to get it could you please help me with this! @fabianocouto @Zo-Bro-23 @rickstaa @anuraghazra
> 
> Hi, @vishalsingh2972! I don't know if I understood your idea very well. But are you suggesting something like that?
> 
> I added a new option called `rank_icon` with initially two possible values: `default` or `github`.
> 
> ```
> https://fabianocouto-readme-stats.vercel.app/api?username=fabianocouto&show_icons=true&include_all_commits=true&count_private=true&theme=github_dark_dimmed&hide=stars&line_height=28&rank_icon=github
> ```
> 
> fabianocouto
> 
> fabianocouto
> 
> fabianocouto

Hey @fabianocouto. The `rank_icon` looks excellent 🚀! Would it be possible to create a PR so that I can merge it into the main branch?

**Pull request merged**